types/jasmine/index.d.ts

```diff
@@ -440,14 +441,16 @@ declare namespace jasmine {
         toThrow(expected?: any): boolean;
         toThrowError(message?: string | RegExp): boolean;
         toThrowError(expected?: new (...args: any[]) => Error, message?: string | RegExp): boolean;
+        nothing(): Any;
+
         not: Matchers<T>;
 
         Any: Any;
     }
 
     interface ArrayLikeMatchers<T> extends Matchers<ArrayLike<T>> {
-        toBe(expected: Expected<ArrayLike<T>>, expectationFailOutput?: any): boolean;
-        toEqual(expected: Expected<ArrayLike<T>>, expectationFailOutput?: any): boolean;
+        toBe(expected: Expected<ArrayLike<T>> | ArrayContaining<T>, expectationFailOutput?: any): boolean;
```

Comment:
`Expected<ArrayLike<T> | ArrayContaining<T>>` ?

Reply:
as mentioned in the linked issue #21305 the `ArrayLikeMatchers` did not accept the type returned by `jasmine. arrayContaining()`

because of that I added the `ArrayContaining<T>` type